tests/integration/shell/test_matcher.py

```diff
@@ -102,25 +102,23 @@ def test_compound_pillar(self):
 
     @skipIf(True, "This test is unreliable. Need to investigate why more deeply.")
     @flaky
-    def test_coumpound_pillar_pcre(self):
+    def test_compound_pillar_pcre(self):
         data = self.run_salt("-C 'J%@knights%^(Lancelot|Galahad)$' test.ping")
         self.assertTrue(minion_in_returns("minion", data))
         self.assertTrue(minion_in_returns("sub_minion", data))
-        # The multiline nodegroup tests are failing in develop.
```

Comment:
If we are laving commented out code, I think we should create an issue to track the work that still needs to be done. We can assign the issue to you.

Reply:
It's the opposite here, I've reinstated some old commented-out tests.